spec/requests/skateparks_spec.rb

```diff
@@ -30,6 +43,17 @@
     expect(json_body['lon']).to eq(other.lon)
   end
 
+  it 'returns users that have favorited skatepark' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.